Vizzuality/waterbird-population
# Feature/info card grid

This PR adds:

New route for detailed data in explore section.
Component to link to detailed data.

Fix some styles and reorganice info components in explore section.

## Testing instructions

Go to http://localhost:3000/explore, click on info card. It should take you to http://localhost:3000/explore/info

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/171327558
